tests/conftest.py

```diff
@@ -48,3 +53,39 @@ def pytest_addoption(parser: _pytest.config.argparsing.Parser) -> None:
     parser.addoption(
         "--no-js", action="store_true", default=False,
         help="only run python code and skip js")
+
+def pandas_constructor(obj) -> IntoDataFrame:
+    return pd.DataFrame(obj)  # type: ignore[no-any-return]
```

Comment:
Thinking out loud here. I believe that we should eventually move these into narwhals (or something like narwhals-testing?). It's the 2nd/3rd time we add such constructors to a repo.

Maybe we could mark [narwhals#739](https://github.com/narwhals-dev/narwhals/issues/739) as medium priority